src/components/typings/OptionGroupCheckBox.ts

```diff
@@ -3,6 +3,7 @@ import { CommonProps, Extras } from "./OptionGroup";
 export interface OptionGroupCheckBoxProps<OptionType> extends CommonProps {
   onChange: (value: OptionType[], extras: Extras) => void;
   selected?: OptionType[];
```

Comment:
add a comment deprecated above this. Then typescript will tell whenever you use it.